gui/tasks/distribution.js

```diff
@@ -284,6 +287,10 @@ function packMac() {
             break;
         }
 
+        process.env.BINARIES_PATH = hostTargetTriple !== process.env.TARGET_TRIPLE
```

Comment:
Nit: Perhaps a parentheses could make this more readable
```suggestion
        process.env.BINARIES_PATH = (hostTargetTriple !== process.env.TARGET_TRIPLE)
```

Reply:
It may just be me who's unused to ternary operators. If you disagree then just leave it as it was.

Reply:
Without parentheses is how the linter wants it :)